### PR TITLE
Add missing element id for aria-labelledby

### PR DESCRIPTION
**Pull request by paulhowes-HMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-2127

### Change description ###
Add missing element id for aria-labelledby on notification banner. Note: In addition, y18n has been added and one test has been temporarily commented out by QA.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Mo-Lala84 — DISMISSED** [review]
lgtm

File: test/integration/tests/utils/getCookie.ts [hunk 1]
```diff
@@ -91,9 +91,9 @@ export async function xxsrftoken()  {
 
 function getPuppeteerLaunchOptions(url) {
   const puppeteerOption = { ignoreHTTPSErrors: true, headless: true, args: [] };
-  if (!url.includes('manage-org.')) {
-    puppeteerOption.args.push('--proxy-server=http://proxyout.reform.hmcts.net:8080');
-  }
+ // if (!url.includes('manage-org.')) {
```
Comment (paul-graham) [inline test/integration/tests/utils/getCookie.ts:94]: Should these lines be deleted rather than commented out?

**Review by paul-graham — COMMENTED** [review]
Just a couple of questions...

**Comment by paul-graham** [comment]
Oh, I asked the other question on Slack: should we still be using `package-lock.json` when we have `yarn.lock`?

File: test/integration/tests/utils/getCookie.ts (hunk 1 above)
Comment (paulhowes-HMCTS) [reply test/integration/tests/utils/getCookie.ts:94]: As per PR description: one test has been temporarily commented out by QA.

Comment (paul-graham) [reply test/integration/tests/utils/getCookie.ts:94]: Ah! My bad; I missed that bit of the description.

**Comment by paulhowes-HMCTS** [comment]
> Oh, I asked the other question on Slack: should we still be using `package-lock.json` when we have `yarn.lock`?

I've removed package-lock.json files from both the app and api folders and am rebuilding the PR.

**Review by Mo-Lala84 — APPROVED** [review]
done. thanks for actioning

**Review by paul-graham — APPROVED** [review]

**Comment by paulhowes-HMCTS** [comment]
Closed as merged in as part of a cumulative release of old PR's

**Comment by paulhowes-HMCTS** [comment]
Closing PR as was merged as part of one of the "several PR's in one" pull requests.

**Pull request closed without merging**